### PR TITLE
#1333 Pipeline graph is not updating correctly when making changes in the config form

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1333 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Mui/Form/InputField.js [hunk 1]
```diff
@@ -48,7 +48,6 @@ const InputField = ({
         {...rest}
         name={name}
         error={meta.error && meta.touched}
-        onChange={onChange}
```
Comment (wu87988622) [inline ohara-manager/client/src/components/common/Mui/Form/InputField.js]: remove onChange will affect other use Mui page

File: ohara-manager/client/src/components/common/Mui/Form/Select.js [hunk 2]
```diff
@@ -45,7 +45,6 @@ const Select = props => {
       name={name}
       error={meta.error && meta.touched}
       inputProps={restInput}
-      onChange={onChange}
```
Comment (wu87988622) [inline ohara-manager/client/src/components/common/Mui/Form/Select.js]: Same as above

**Comment by wu87988622** [comment]
retry manager

File: ohara-manager/client/src/components/common/Mui/Form/InputField.js (hunk 1 above)
Comment (konekoya) [reply ohara-manager/client/src/components/common/Mui/Form/InputField.js]: You're right. I haven't figure out why this is happening yet. But a temp workaround is here to make sure that the UI will behave as it should. I'll need more time to investigate this behavior :(

File: ohara-manager/client/src/components/common/Mui/Form/Select.js (hunk 2 above)
Comment (konekoya) [reply ohara-manager/client/src/components/common/Mui/Form/Select.js]: Addressed with a temp solution

File: ohara-manager/client/src/components/common/Mui/Form/Tooltip.js [hunk 3]
```diff
@@ -34,6 +34,9 @@ const StyledIcon = styled(InfoIcon)`
 `;
 
 const Tooltip = ({ text }) => {
+
+  if (!text) return null
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Mui/Form/Tooltip.js]: Make sure we're rendering tooltip in other components

**Comment by konekoya** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by konekoya** [comment]
Go go go!

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

**Pull request merged**